SubjectsDAL/Models/Human.cs

```diff
@@ -8,5 +8,6 @@ namespace SubjectsDAL.Models
 {
     public class Human : Person
     {
+
```

Comment:
You should consider removing this empty class and make for example an interface.